docs/reference/settings/data-frames-settings.asciidoc

```diff
@@ -0,0 +1,39 @@
+
+[role="xpack"]
+[[data-frames-settings]]
+=== {dataframe-transforms-cap}  settings in Elasticsearch
+++++
+<titleabbrev>{dataframe-transforms} settings</titleabbrev>
+++++
+
+You do not need to configure any settings to use {dataframe-transforms}. It is enabled by default.
+
+All of these settings can be added to the `elasticsearch.yml` configuration file. 
+The dynamic settings can also be updated across a cluster with the 
```

Comment:
Since neither of these settings seem to be dynamic, I think we should comment out "The dynamic settings..." and the "TIP: Dynamic settings..." content.  
If one of these settings *is* dynamic, we should add "(Dynamic)" after its name (like xpack.ml.max_open_jobs, for example, in https://www.elastic.co/guide/en/elasticsearch/reference/master/ml-settings.html).